syft/serde/serde.py

```diff
@@ -114,6 +114,13 @@
 
 ## SECTION: High Level Simplification Router
 def _force_full_simplify(obj: object) -> object:
+    """
+    To force a full simplify genrally if the usual _simplify is not suitable.
+    
+    Args:
+    :param: the object
```

Comment:
Hey, can you please switch this to be a [Google style docstring](http://google.github.io/styleguide/pyguide.html#383-functions-and-methods)? :)

Reply:
Sure